libr/core/cmd_anal.c

```diff
@@ -8521,7 +8708,7 @@ static void cmd_anal_graph(RCore *core, const char *input) {
 		case 'w':
 		case ' ': {
 			core->graph->is_callgraph = true;
-			r_core_cmdf (core, "ag-; .agc* @ %"PFMT64u"; agg%s;", core->offset, input + 1);
+			r_core_cmdf (core, "ag-; .agc* @ %" PFMT64u "; agg%s;", core->offset, input + 1);
```

Comment:
Code injection

Reply:
*) was there before this issue
*) might get removed in the follow up issues which convert more graphs to the mechanism introduce in this issue